Realm/RLMLogger.mm

```diff
@@ -140,13 +208,44 @@ - (void)logWithLevel:(RLMLogLevel)logLevel message:(NSString *)message, ... {
     }
 }
 
-- (void)logLevel:(RLMLogLevel)logLevel message:(NSString *)message {
+- (void)logWithLevel:(RLMLogLevel)logLevel category:(RLMLogCategory)category message:(NSString *)message {
     auto level = levelForLogLevel(logLevel);
     if (_logger->would_log(level)) {
```

Comment:
It is not filtering by category, just by level. There is an alternative signature for `would_log` that takes the category as a parameter.